OsmAnd/src/net/osmand/plus/activities/FavoritesTreeFragment.java

```diff
@@ -752,8 +760,31 @@ void synchronizeGroups() {
 			for (FavoriteGroup key : gs) {
 				boolean empty = true;
 				if (flt == null || flt.contains(key)) {
-					empty = false;
-					favoriteGroups.put(key, new ArrayList<>(key.points));
+					if (key.personal) {
+						ArrayList<FavouritePoint> list = new ArrayList<>();
+						for (FavouritePoint p : key.points) {
+							if (p.getName().equals(FavouritePoint.PointType.PARKING.getName())) {
```

Comment:
1. it is wrong place to customize "isParkingPluginEnable" should not be here in the code
2. there is no need fofr this ifs , it should be always visible 
Code should be deleted

Reply:
What if parking plugin will be disabled? We must hide parking point?